dashboard/config/routes.rb

```diff
@@ -759,6 +759,7 @@
             patch :bulk_update_owners
           end
         end
+        resources :deep_linking, only: :index
```

Comment:
I would suggest using `resource` with a `show` action since the routes represent a singular object:

```suggestion
        resource :deep_linking, controller: :deep_linking, only: :show
```

```bash
            Prefix Verb URI Pattern             Controller#Action
deep_linking_index GET  /deep_linking(.:format) deep_linking#index
      deep_linking GET  /deep_linking(.:format) deep_linking#show
```